enclave_manager/Dockerfile

```diff
@@ -141,7 +141,7 @@ WORKDIR /tmp
 
 RUN ldconfig \
  && ln -s /etc/ssl/certs/* /usr/local/ssl/certs/ \
- && git clone https://github.com/intel/intel-sgx-ssl.git  \
```

Comment:
Add a comment as using the tag “lin_2.5_1.1.1d” as we are using openssl 1.1.1d version.

Reply:
Done